### ISSUE TITLE
Memory Leak?

### ISSUE THREAD

**Issue by gentleman991** [ccd0/4chan-x#1540]
Getting an extra 1 gig of memory use compared to when the script is disabled on firefox 56.0.1.

**Comment by ccd0** [comment]
What version of Greasemonkey?

**Comment by gentleman991** [comment]
3.17

**Comment by ccd0** [comment]
Do you have any features like "Replace ___" or "Auto-embed" turned on? Does it happen when you disable other extensions?

**Comment by lc-guy** [comment]
Same here. Dunno what's up.

**Comment by lc-guy** [comment]
More specifically, it seems to happen while I'm typing...

**Comment by ccd0** [comment]
If I understand you right, memory usage goes up dramatically as you type in the Quick Reply?

**Comment by lc-guy** [comment]
Yep. No extension that changes writing behavior enabled either. Keep in
mind I'm not OP.

**Comment by ccd0** [comment]
Can you try commenting out some/all of the lines in `updateComment` to see if it has any effect?

```
    _Class.prototype.updateComment = function() {
      if (this === QR.selected) {
        QR.characterCount();
      }
      this.nodes.span.textContent = this.com;
      return QR.captcha.moreNeeded();
    };
```

This gets executed whenever you type in the QR.

**Comment by lc-guy** [comment]
At least it doesn't outright freeze my computer, but the typing is laggy.
Might actually be Firefox-related, since it started happening when I upgraded to 56. I'll wait until 57 releases and then comment on whether or not it is still happening.
